src/controllers/user/send-recovery-email.js

```diff
@@ -0,0 +1,44 @@
+const nodemailer = require('nodemailer');
+const crypto = require('crypto');
+const getUser = require('../user/get-user');
+const updateUser = require('../user/update-user');
+const WEB_APP_BASE_URL = require('../../settings');
+
+const sendRecoveryEmail = async (username) => {
+  try {
+    await getUser({ username });
+
+    const token = crypto.randomBytes(20).toString('hex');
+    const { EMAIL_ADDRESS, EMAIL_PASSWORD } = process.env;
```

Comment:
`EMAIL_ADDRESS` and `EMAIL_PASSWORD` are too ambiguous as environment variables. From what I can see, these are used as the source email address from which system emails are sent. Perhaps you could use more descriptive names for these vars.

Reply:
You are right. I have updated these variable names to provide better description.